Source/SwiftLintFramework/Rules/TrailingNewlineRule.swift

```diff
@@ -8,7 +8,19 @@
 
 import SourceKittenFramework
 
-public struct TrailingNewlineRule: Rule {
+extension String {
+    public func trailingNewlineCount() -> Int? {
```

Comment:
This could probably be `private` unless you can think of another place we might use it.